src/worker/tasks/sendTransactionWorker.ts

```diff
@@ -106,9 +108,27 @@ const handler: Processor<any, void, string> = async (job: Job<string>) => {
       // Report usage only on the first transaction send.
       if (transaction.status === "queued") {
```

Comment:
interesting that this is not `resultTransaction`?

Reply:
`transaction.status: "queued" | "sent"`
`resultTransaction.status: "sent" | "errored"`